lib/mutant/mutator/node/binary.rb

```diff
@@ -60,6 +61,21 @@ def emit_left_negation
           emit(s(node.type, n_not(left), right))
         end
 
+        # Emit mutation from `a[b] || c` to `a.fetch(b) { c }`
+        #
+        # @return [undefined]
+        def emit_fetch_mutation
+          send_node = AST::Meta::Send.new(left)
```

Comment:
Mutant kept removing it and I'm not sure I can think of another node that might have `:[]` as the second positional argument in the node (making it the `#selector` attribute under the meta)